soc/nxp/imx/imx9/imx95/m7/linker.ld

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/arm/cortex_m/scripts/linker.ld>
```

Comment:
we can just include the architecture linker script directly using CMakeLists.txt. Example: https://github.com/zephyrproject-rtos/zephyr/pull/77228.

Reply:
Okay, thanks!